extensions/sql-database-projects/src/models/tree/sqlcmdVariableTreeItem.ts

```diff
@@ -64,6 +64,7 @@ export class SqlCmdVariablesTreeItem extends BaseProjectTreeItem {
 export class SqlCmdVariableTreeItem extends BaseProjectTreeItem {
 	constructor(private sqlcmdVar: string, sqlCmdNodeRelativeProjectUri: vscode.Uri, sqlprojUri: vscode.Uri,) {
 		super(vscode.Uri.file(path.join(sqlCmdNodeRelativeProjectUri.fsPath, sqlcmdVar)), sqlprojUri);
+		this.entryKey = this.friendlyName;
```

Comment:
should database reference tree item also have `entryKey` set so all the project's child nodes that can be added/deleted have it set?